### PR TITLE
Update garment details to show sizes, category, type, status and licence

### PR DESCRIPTION
**Pull request by Bramvanosta**

### CODE REVIEW

File: src/components/pages/Create.vue [hunk 1]
```diff
@@ -36,12 +43,14 @@ export default {
       categories: ['Kid', 'Teen', 'Adult', 'Mature', 'MILF'],
       sizes: ['XS', 'S', 'M', 'L', 'XL'],
       types: ['T-shirt', 'Sweat', 'Pants', 'Accessories', 'Underwear'],
+      licences: ['CC BY', 'CC BY-SA', 'CC BY-ND', 'CC BY-NC', 'CC BY-NC-SA', 'CC BY-NC-ND', 'CC0', 'none'],
```
Comment (DavidBruant) [inline src/components/pages/Create.vue]: Use constants (maybe in their own file)

File: src/components/pages/Create.vue [hunk 2]
```diff
@@ -36,12 +43,14 @@ export default {
...
       garment: {
         title: '',
         category: '',
         sizes: [],
         description: '',
         type: '',
+        licence: 'CC BY',
```
Comment (DavidBruant) [inline src/components/pages/Create.vue]: use one of the constants

File: src/components/pages/Create.vue [hunk 3]
```diff
@@ -17,6 +19,10 @@
             </div>
             <label for="description">Description</label>
             <textarea name="description" id="description" v-model="garment.description"></textarea>
+            <label for="licence">Licence</label>
+            <select name="licence" id="licence" v-model='garment.licence'>
+                <option v-for="licence in licences" :value="licence">{{licence}}</option>
```
Comment (DavidBruant) [inline src/components/pages/Create.vue:24]: The default licence should be `selected` here

File: src/components/pages/Create.vue [hunk 4]
```diff
@@ -51,33 +60,41 @@ export default {
...
-          return remoteRepo.writeFile('master', garmentConfig, JSON.stringify(obj), 'Garment project setup', function () {})
-          .then(function (repoResponse) {
-            console.log('success');
-            console.log(repoResponse);
+        .then(function (profileResponse) {
+          const user = profileResponse.data.login;
+          return gh.getUser(user).createRepo(options).then(function (createResponse) {
+            const repo = createResponse.data.name;
+            const remoteRepo = gh.getRepo(user, repo);
+            return remoteRepo.writeFile('master', garmentConfig, JSON.stringify(garmentConfigOptions), 'Garment project setup', function () {})
```
Comment (DavidBruant) [inline src/components/pages/Create.vue]: why the empty callback?

**Review by DavidBruant — CHANGES_REQUESTED** [review]
See comments. LGTM otherwise

File: src/components/pages/Create.vue (hunk 3 above)
Comment (Bramvanosta) [reply src/components/pages/Create.vue:24]: By setting the default value of `garment.licence`, the option corresponding to that licence will be `selected` by default.

Comment (DavidBruant) [reply src/components/pages/Create.vue:24]: 👍

File: src/components/pages/Create.vue (hunk 4 above)
Comment (Bramvanosta) [reply src/components/pages/Create.vue]: Empty callback/object is necessary for the API call to work (represents the option parameter).

File: src/components/pages/Create.vue [hunk 5]
```diff
@@ -36,12 +44,22 @@ export default {
...
       sizes: ['XS', 'S', 'M', 'L', 'XL'],
       types: ['T-shirt', 'Sweat', 'Pants', 'Accessories', 'Underwear'],
+      licences: [
+        constants.CC_BY,
+        constants.CC_BY_SA,
+        constants.CC_BY_ND,
+        constants.CC_BY_NC,
+        constants.CC_BY_NC_SA,
+        constants.CC_BY_NC_ND,
+        constants.CC0,
```
Comment (DavidBruant) [inline src/components/pages/Create.vue:54]: The `none` case is missing

**Review by DavidBruant — CHANGES_REQUESTED** [review]

**Pull request merged**